### PR TITLE
Enable URL on PHP resource application

### PR DESCRIPTION
**Pull request by kiuby88**

### CODE REVIEW

**Comment by kiuby88** [comment]
@adriannieto, take a look please

File: deployer/src/test/java/eu/seaclouds/location/cloudfoundry/PhpWebCloudFoundryLiveTest.java [hunk 1]
```diff
@@ -39,9 +39,7 @@
 
 public class PhpWebCloudFoundryLiveTest extends AbstractCloudFoundryPaasLocationLiveTest {
 
-    private final String APPLICATION_URL = checkNotNull(getClass().getClassLoader()
-            .getResource("phpHelloWorld")).getFile();
-
+    private final String APPLICATION_URL = "https://github.com/kiuby88/phpHelloWorld/releases/download/1/phpHelloWorld.zip";
```
Comment (perezp) [inline deployer/src/test/java/eu/seaclouds/location/cloudfoundry/PhpWebCloudFoundryLiveTest.java]: Is there any reason for not having the zip file in some test resources folder inside https://github.com/SeaCloudsEU/SeaCloudsPlatform/....  ??

Comment (kiuby88) [inline deployer/src/test/java/eu/seaclouds/location/cloudfoundry/PhpWebCloudFoundryLiveTest.java]: @perezp sorry for my late response
I have add a .zip to test resources

**Comment by perezp** [comment]
Looks good.

**Pull request merged**